src/locale/locales/fr/messages.po

```diff
@@ -5203,39 +4839,31 @@ msgstr "Désépingler"
 
 #: src/view/screens/ProfileFeed.tsx:303
 msgid "Unpin from home"
-msgstr ""
+msgstr "Désépingler de l’accueil"
 
 #: src/view/screens/ProfileList.tsx:446
 msgid "Unpin moderation list"
 msgstr "Supprimer la liste de modération"
 
-#: src/view/screens/ProfileFeed.tsx:346
-#~ msgid "Unsave"
-#~ msgstr "Supprimer"
-
 #: src/screens/Profile/Header/ProfileHeaderLabeler.tsx:225
 msgid "Unsubscribe"
-msgstr ""
+msgstr "Désabonner"
 
 #: src/screens/Profile/Header/ProfileHeaderLabeler.tsx:190
 msgid "Unsubscribe from this labeler"
-msgstr ""
+msgstr "Désabonner de cet étiqueteur"
 
 #: src/lib/moderation/useReportOptions.ts:70
 msgid "Unwanted Sexual Content"
-msgstr ""
+msgstr "Contenu sexuel non désiré"
 
 #: src/view/com/modals/UserAddRemoveLists.tsx:70
 msgid "Update {displayName} in Lists"
 msgstr "Mise à jour de {displayName} dans les listes"
 
-#: src/lib/hooks/useOTAUpdate.ts:15
-#~ msgid "Update Available"
-#~ msgstr "Mise à jour disponible"
-
 #: src/view/com/modals/ChangeHandle.tsx:508
 msgid "Update to {handle}"
-msgstr ""
+msgstr "Mise à jour à {handle}"
```

Comment:
We `mettre à jour à` for a version of a software, but if we update as a fancy way of saying `change`, we `mettre à jour pour` something.

```suggestion
msgstr "Mettre à jour pour {handle}"
```